<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunnerChartsDialog</class>
 <widget class="QDialog" name="RunnerChartsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Runner Charts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout" stretch="12,1">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="chartLayout"/>
     </item>
     <item>
      <widget class="QGroupBox" name="gbOptions">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>80</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox#gbOptions {
     border: 1px solid gray;
}
</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="lyRunnerInfo" stretch="1,1,1,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="lblChooseRunner">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Selected Runner </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbChartRunnerSelect">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblUtil1">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblUtil2">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="lyXAxisOptions" stretch="1,1,1,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>X-Axis Options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbXAxisDisplay">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="lyYAxisOptions" stretch="1,1,1,1">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Y-Axis Options</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbYAxisDisplay">
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>7</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
